icubam/backoffice/handlers/users.py

```diff
@@ -39,50 +40,111 @@ def get(self):
       '{}?id={}'.format(UserHandler.ROUTE, self.user.user_id))
 
 
-
 class UserHandler(base.BaseHandler):
   ROUTE = "/user"
 
+  """Creates ICU DTOs based on ALL ICUs and those from the subset."""
+  def prepare_icus(self, subset_icus_ids):
+    icus = self.store.get_icus()
```

Comment:
there should be a slight difference between admin and managers. But let's have a todo here `TODO(olivier): restrict managers to regional icus`

Reply:
Added.